tests/integration/pipeline.ts

```diff
@@ -462,6 +460,38 @@ describe('Pipeline Editor tests', () => {
     cy.readFile('build/cypress-tests/helloworld.yaml');
   });
 
+  it('should export pipeline as python dsl', () => {
+    // Create runtime configuration
+    cy.createRuntimeConfig({ type: 'kfp' });
+
+    // go back to file browser
+    cy.findByRole('tab', { name: /file browser/i }).click();
+
+    cy.openFile('helloworld.pipeline');
+
+    // try to export valid pipeline
+    cy.findByRole('button', { name: /export pipeline/i }).click();
+
+    cy.findByLabelText(/runtime configuration/i)
+      .select('test_runtime') // there might be other runtimes present when testing locally, so manually select.
+      .should('have.value', 'test_runtime');
+
+    // Validate all export options are available
+    cy.findByLabelText(/export pipeline as/i)
+      .select('KFP domain-specific language Python code')
+      .should('have.value', 'py');
+
```

Comment:
This should do this trick:
```suggestion

cy.findByLabelText(/replace if file already exists/i)
      .check()
      .should('be.checked');
```